derive_client/clients/base_client.py

```diff
@@ -436,6 +387,14 @@ def fetch_ticker(self, instrument_name):
         results = json.loads(response.content)["result"]
         return results
 
+    def fetch_tickers(
+        self,
+        instrument_type: InstrumentType = InstrumentType.OPTION,
+        currency: UnderlyingCurrency = UnderlyingCurrency.BTC,
+    ):
+        instruments = self.fetch_instruments(instrument_type=instrument_type, currency=currency)
+        return {inst["instrument_name"]: self.fetch_ticker(inst["instrument_name"]) for inst in instruments}
+
```

Comment:
Refactored function (from using WS to REST) to appear immediately after the singular version it builds on. Replaces the deleted version below for clearer logical grouping

Reply:
I am almost 100% sure this will get rate limited a lot.
I recall this being for somereason quite an expensive request